### DISCUSSION TITLE
system keeps saying in using 100% resorces

### DISCUSSION THREAD

**Discussion by DMTarmey** [community/community#59994] [Codespaces]
### Select Topic Area

General

### Body

So if I'm using 100% resources how can i get bigger account.

**Comment by dmilburn** [comment]
Hi @DMTarmey,

By 100% of resources, do you mean the resources of the VM itself? Or do you mean the ability to create/use a Codespace?

If you mean VM resources:

- To request a larger machine type, please make a request to GitHub's official support team by filing a ticket here: https://support.github.com/contact?tags=rr-codespaces%2Ccat_codespaces

If you mean creating/using a Codespace:

- It looks to me like you may have already solved this problem yourself by creating a spending limit since you surpassed the amount of Codespaces usage included by your account. But please let me know if I misunderstood
